libs/common-types/src/lib.rs

```diff
@@ -63,9 +100,28 @@ bitflags::bitflags! {
 		const LIQUIDITY_ADMIN = 0b00001000;
```

Comment:
@mustermeiszer wdyt, should we change this to `u64` already to accommodate more roles in the future?

Reply:
It's twice the storage. Do you think we will have more than 32 roles on a pool level? We can add those bitflags for other role locations too. But if we think we will have more than 32 roles, better adapt now!

Reply:
Yeah you're right, 32 should be fine.